packages/foam-core/src/model/position.ts

```diff
@@ -0,0 +1,87 @@
+export interface Position {
+  line: number;
+  character: number;
+}
+
+export const create = (line: number, character: number): Position => ({
```

Comment:
Instead of having a separate named export re-exported from index, we can do:
```ts
export default interface Position {
  line: number;
  character: number;
}

export default abstract class Position implements Position {
  static create(line: number, character: number): Position {
    return {
      line,
      character
    };
  }
}
```